app/views/admin/financial_reports/confirm_destroy.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :context, "Financial report" %>
```

Comment:
I suspect this should be

```
<% content_for :context, @organisation.name %>
```

might be worth double checking with Nik, but that's generally the pattern we've followed.

Reply:
> I suspect this should be
> 
> ```
> <% content_for :context, @organisation.name %>
> ```
> 
> might be worth double checking with Nik, but that's generally the pattern we've followed.

Yes actually that would make more sense, and be more consistent. 👍